.pre-commit-config.yaml

```diff
@@ -112,32 +112,6 @@ repos:
       - id: debug-statements
       - id: name-tests-test
         args: [--pytest-test-first]
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: 'v3.0.3' # Use the sha or tag you want to point at
-    hooks:
-      - id: prettier
-        name: prettier (yaml, json, markdown)
-        types_or:
-          - json
-          - yaml
-          - markdown
```

Comment:
does biome format json / yaml / markdown? I was under the impression it was mainly targetted at javascript

Reply:
Here's a list of supported languages: https://biomejs.dev/internals/language-support/